src/main/fc/settings.yaml

```diff
@@ -219,13 +219,13 @@ groups:
         default_value: 1000
         max: 9000
       - name: gyro_anti_aliasing_lpf_hz
-        description: "Gyro processing anti-aliasing filter cutoff frequency. In normal operation this filter setting should never be changed. In Hz"
-        default_value: 250
+        description: "Gyro processing anti-aliasing filter cutoff frequency. In normal operation this filter setting should never be changed. In samples"
+        default_value: 1
```

Comment:
Let's not run lulu on antialiasing filter at all. It makes no sense as this filter is only to remove aliasing effect between acquisition and processing